deploy/charts/fedlearner-add-on/values.yaml

```diff
@@ -3,32 +3,35 @@
 # Declare variables to be passed into your templates.
 
 imageCredentials:
-  enabled: false
+  enabled: true
   registry: ""
   username: ""
   password: ""
 
 service:
-  enabled: false
+  enabled: true
   type: ExternalName
   externalName: ""
 
 ingress:
-  enabled: false
+  enabled: true
   annotations:
+    "nginx.ingress.kubernetes.io/proxy-body-size": 10g
+    "nginx.ingress.kubernetes.io/backend-protocol": GRPCS
+    "nginx.ingress.kubernetes.io/http2-insecure-port": t
```

Comment:
change t to true, t is a workaround to pass params to yaml

Reply:
fixed